devito/memory.py

```diff
@@ -8,10 +8,9 @@
 import numpy as np
 from sympy import Eq
 
-from devito.dimension import t
 from devito.logger import error
 from devito.tools import convert_dtype_to_ctype
-
+import devito
```

Comment:
Any idea of how we could avoid the circular dependencies that we would have with ``from devito import Operator`` ?

I think this is a symptom that we should do things differently